<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFFR</class>
 <widget class="QMainWindow" name="frmFFR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>方向图设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCreate">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>起始</string>
    </property>
    <widget class="QLabel" name="lblThe_start">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>θ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>φ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtThe_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhi_start">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxEnd">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>终止</string>
    </property>
    <widget class="QLabel" name="lblThe_end">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>θ</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>φ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtThe_end">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhi_end">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxIncrement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>步进</string>
    </property>
    <widget class="QLabel" name="lblThe_inc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>θ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhi_inc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>φ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtThe_inc">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhi_inc">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxNum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>观察点数</string>
    </property>
    <widget class="QLabel" name="lblThe_n">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>θ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPhi_n">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>φ</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtThe_n">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhi_n">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>126</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRequestName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>名称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFreq">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>241</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRequestName_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>频率(Hz)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
